contracts/PolygonTokenBridger.sol

```diff
@@ -38,7 +38,7 @@ interface MaticToken {
  *  This ultimately allows create2 to generate deterministic addresses that don't depend on the transaction count of the
  * sender.
  */
-contract PolygonTokenBridger is Lockable {
+contract PolygonTokenBridger is ReentrancyGuard {
```

Comment:
Slither had some complaints about Lockable but i figured this was a good time to use OZ's ReentrancyGuard instead

Reply:
I don't think this contract will be in scope for the audit. Are we okay making this change?

Reply:
good point, removed